blivet/devicefactory.py

```diff
@@ -52,14 +54,27 @@
 SIZE_POLICY_MAX = -1
 SIZE_POLICY_AUTO = 0
 
-DEVICE_TYPE_LVM = 0
-DEVICE_TYPE_MD = 1
-DEVICE_TYPE_PARTITION = 2
-DEVICE_TYPE_BTRFS = 3
-DEVICE_TYPE_DISK = 4
-DEVICE_TYPE_LVM_THINP = 5
-DEVICE_TYPE_LVM_VDO = 6
-DEVICE_TYPE_STRATIS = 7
+
+@unique
+class DEVICE_TYPES(Enum):
+    """
+    Types of devices that blivet is aware of.
```

Comment:
These are device types supported by the device factory, not all devices blivet supports.

Reply:
Docstr5ing updated.